src/libraries/System.IO.FileSystem/src/Resources/Strings.resx

```diff
@@ -197,6 +197,9 @@
   <data name="IO_CannotCreateDirectory" xml:space="preserve">
     <value>The specified directory '{0}' cannot be created.</value>
   </data>
+  <data name="IO_CannotReplaceSameFile" xml:space="preserve">
+    <value>The source and destination are the same file.</value>
```

Comment:
Include path? These are often in logs, where that can be helpful

Reply:
I am slowly working to add paths to all IO messages where we have a path available to include...

Reply:
I'll add the paths in the message.